.github/ci-scripts/bash/functions-for-build

```diff
@@ -263,6 +269,26 @@ frp_build_and_push() {
   set +e
 }
 
+gcp_docker_build() {
```

Comment:
For `tessell`, we have a file `setup` in each golang based repo, which we use for build.
Check if we can re-use that instead of having a new function (`gcpDockerBuild`)